src/monitoring/universal_logs_listener.py

```
@@ -0,0 +1,212 @@
+            self.platforms = platform_factory.get_supported_platforms()
+        else:
+            self.platforms = platforms
+            
+        # Get event parsers for all platforms
+        self.platform_parsers = {}
+        self.platform_program_ids = []
+        
+        for platform in self.platforms:
+            try:
+                # Create a simple dummy client that doesn't start blockhash updater
+                from core.client import SolanaClient
```

Ruff (0.12.2)

`import` should be at the top-level of a file

(PLC0415)